db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/InntektsmeldingRepository.kt

```diff
@@ -36,82 +38,98 @@ class InntektsmeldingRepository(
         forespoerselId: UUID,
         inntektsmeldingDokument: Inntektsmelding,
     ) {
-        val requestTimer = requestLatency.labels("lagreInntektsmelding").startTimer()
-        transaction(db) {
-            InntektsmeldingEntitet.insert {
-                it[this.forespoerselId] = forespoerselId.toString()
-                it[dokument] = inntektsmeldingDokument
-                it[innsendt] = LocalDateTime.now()
+        requestLatency.recordTime(label = "lagreInntektsmelding") {
+            transaction(db) {
+                InntektsmeldingEntitet.insert {
+                    it[this.forespoerselId] = forespoerselId.toString()
+                    it[dokument] = inntektsmeldingDokument
+                    it[innsendt] = LocalDateTime.now()
```

Comment:
Denne bør kanskje bli en update, siden man inserter skjema først?

Reply:
Hmm, jo, men jeg lurer på om vi kanskje må være litt ekstra lure for å vite hvilken (av de potensielt flere) skjemaene vi trenger å oppdatere med IM-dokumentet 🤔 Ref. det vi diskuterte tidligere rundt at det kan være innsender har sendt inn et nytt skjema før vi har rukket å berike det første skjemaet (f.eks. dersom PDL er nede og vi må vente til FeilLytter sparker i gang berikelsen igjen).